packages/flight-icons/scripts/build-parts/generateBundleCSSProps.ts

```diff
@@ -0,0 +1,47 @@
+import fs from 'fs-extra';
+
+import { ConfigData } from '../@types/ConfigData';
+import { AssetsCatalog } from '../@types/AssetsCatalog';
+import { getCssForIconAnimation } from './getCssForIconAnimation';
+
+const getCssProperty = ({ fileName, svgSource }: { fileName: string; svgSource: any }) => {
+
+    // remove width/height attributes
+    svgSource = svgSource.replace(/ width="\d+" height="\d+"/gi, '');
+
+    // replace #000001 ("dynamic" color in Figma) with #000 (the icon needs to be tinted with other ways)
+    svgSource = svgSource.replace(/#000001/g, '#000');
+
+    // replace # with equivalent url-escaped value (no need for full "encodeURIComponent" transformation)
+    svgSource = svgSource.replace(/#/g, "%23");
+
+    return `--icon-${fileName}: url('data:image/svg+xml;charset=UTF-8,${svgSource}');`;
+};
+
+export async function generateBundleCSSProps({ config, catalog } : { config: ConfigData, catalog: AssetsCatalog }): Promise<void> {
```

Comment:
This is perhaps a larger question about the flight-icons build pipeline in general, but have you thought at all about how we might create tests for this?  How do we know that future builds of these css props work as intended?

Reply:
most of these operation generate files, so this would mean mock read/write operations, and other funny stuff :)
so the only way to test is to run and see what happens (generally, you expect only new - added or modified - files to change)